.github/workflows/deploy.yaml

```diff
@@ -13,26 +13,40 @@ jobs:
     environment: production
     env:
       CURSORLESS_REPO_ROOT: ${{ github.workspace }}
+
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout repository
+        uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - run: corepack enable
-      - uses: actions/setup-node@v4
+
+      - name: Enable Corepack
+        run: corepack enable
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
         with:
           node-version-file: .nvmrc
           cache: pnpm
-      - run: pnpm --color install
-      - run: pnpm --color compile
-      - run: pnpm --color --filter '!cursorless-org' --filter '!cursorless-org-*' build
+
+      - name: Install dependencies
+        run: pnpm --color install
+
+      - name: Compile
+        run: pnpm --color compile
+
+      - name: Build
+        run: pnpm --color --filter '!cursorless-org' --filter '!cursorless-org-*' build
         env:
```

Comment:
https://github.com/cursorless-dev/cursorless/issues/2391

It makes sense to run compile separately if we think it's faster / higher signal/ROI to read. There's some question about whether `build` would do everything necessary if compile was skipped, which is a bit concerning, but we did not test this.